backends/config/config_test.go

```diff
@@ -155,6 +155,13 @@ func TestNewBaseBackend(t *testing.T) {
 				{msg: "Error creating Redis backend: ", lvl: logrus.FatalLevel},
 			},
 		},
+		//{
```

Comment:
Can this commented out code be deleted?

Reply:
It probably needs to be uncommented, and perhaps worked on if it isn't in working condition.

Reply:
Corrected